cmd/short/short.go

```diff
@@ -10,26 +10,57 @@ import (
 	"net/http"
 	"net/url"
 	"bytes"
+	//"go/build"
+	//"os"
+	"sync"
 )
 
 const CleanUrl_api string = "https://cleanuri.com/api/v1/shorten"
 const Relink_api string = "https://rel.ink/api/links/"
 
 var flagRelink *bool = flag.Bool("relink", false, "a bool")
```

Comment:
Change a help string for the flag. E.g. 
```golang
var flagRelink *bool = flag.Bool("relink", false, "use rel.ink service to shorten URL")
```
This help is printed when you run your app with `-help` flag. e.g:
```sh
$ ./short -help
Usage of ./short:
  -relink
    	use rel.ink service to shorten URL
```